ios: bump Fastlane from 2.205.2 to 2.209.1 and pods

jakubgs opened the pull request:
Also refresh Xcode project integration and fix order of shell hooks for iOS build shell. NodeJS setup needs to be before Pods.

~~Possible fix for errors like:~~ These errors were fixed by deleting `~/Library/Keychains/fastlane.keychain-db` on `maci7-01`:
```
bundler: failed to load command: fastlane (/Users/jenkins/.bundle/ruby/2.7.0/bin/fastlane)
/Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': [!] Shell command exited with exit status 51 instead of 0. (FastlaneCore::Interface::FastlaneShellError)
	from /Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
	from /Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane/lib/fastlane/helper/sh_helper.rb:80:in `sh_control_output'
	from /Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane/lib/fastlane/helper/sh_helper.rb:12:in `sh'
	from /Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane/lib/fastlane/actions/create_keychain.rb:47:in `run'
	from Fastfile:82:in `initialize'
	from Fastfile:125:in `new'
	from Fastfile:125:in `build_ios_adhoc'
	from Fastfile:232:in `block (2 levels) in parsing_binding'
        ...
```
https://ci.status.im/job/status-mobile/job/prs/job/ios/job/PR-13868/6/

jakubgs committed:
ios: bump Fastlane from 2.205.2 to 2.209.1 and pods

Also refresh Xcode project integration.

Signed-off-by: Jakub Sokołowski <jakub@status.im>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 457c4311 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-13884/1/) | 2022-08-29 16:14:49 | ~2 min | `tests` | [:package:`log`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-220829-161244-457c43-pr13884-tests.log) |

jakubgs committed:
nix: fix order of iOS shell components

Possible fix for errors like:
```
bundler: failed to load command: fastlane (/Users/jenkins/.bundle/ruby/2.7.0/bin/fastlane)
/Users/jenkins/.bundle/ruby/2.7.0/gems/fastlane-2.205.2/fastlane_core/lib/fastlane_core/ui/interface.rb:153:in `shell_error!': [!] Shell command exited with exit status 51 instead of 0. (FastlaneCore::Interface::FastlaneShellError)
```

Signed-off-by: Jakub Sokołowski <jakub@status.im>
